src/components/Button/Button.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface ButtonProps {
+    onClick?: () => void;
+    disabled?: boolean;
+    className?: string;
+    imgSrc?: string;
+    imgAlt?: string;
+    imgWidth?: number;
+    imgHeight?: number;
+    children: React.ReactNode;
+}
```

Comment:
instead suggest u to use
```suggestion
interface ButtonProps extends React.ComponentProps<'button'>  {
    src?: string;
    alt?: string;
}
```